lib/OpenQA/Command.pm

```diff
@@ -132,7 +133,8 @@ sub retry_tx ($self, $client, $tx, $retries = undef, $delay = undef) {
 "Request failed, hit error $res_code, retrying up to $retries more times after waiting … (delay: $delay; waited ${waited}s)\n"
         );
         sleep $delay;
-    }
+        $delay = $delay << 1 unless $delay > $max_delay;
```

Comment:
This doesn't seem to include the suggestions yet to use a simple multiplication, ditch the check for maximum, use a configuration value, only exponential on request

Reply:
it was not my intention but i guess i cant fight it.